cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1399,3 +1399,28 @@ declare interface WorkerOptions {
     };
   };
 }
+
+declare interface WebSocketStreamOptions {
+  protocols?: string[];
```

Comment:
```suggestion
  protocols?: string | string[];
```
If I'm not mistaken?

Reply:
nope, just `string[]` for WebSocketStream. The IDL is described in the first resource link

Reply:
I had some trouble finding the idl file in the chromium source tree but I finally dug it up and you're right (and it's weirdly inconsistent with the `WebSocket` constructor, IMO, but that aside):

https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/modules/websockets/websocket_stream_options.idl